bondpy/bondpy/bondpy.py

```diff
@@ -304,7 +307,7 @@ def set_broken_callback(self, on_broken):
     ## \brief Blocks until the bond is formed for at most 'duration'.
     #
     # \param timeout Maximum duration to wait.  If None then this call will not timeout.
-    # \return true iff the bond has been formed.
+    # \return true if the bond has been formed.
```

Comment:
```suggestion
    # \return true iff the bond has been formed.
```

(`iff` means "if and only if")

Reply:
Changed in [74805b1](https://github.com/ros/bond_core/pull/89/commits/74805b1f61bde433b7f39eaf3db5ecf52190c80f)